copilot/declarative-agent/v1.4/schema.json

```diff
@@ -460,6 +625,56 @@
                 }
             }
         },
+        "behavior-overrides": {
+            "type": "object",
+            "description": "An object that contains the behavior overrides for the declarative agent.",
+            "properties": {
+                "special_instructions": {
+                    "description": "An object that contains special instructions for the declarative agent.",
+                    "$ref": "#/$defs/special-instructions"
+                },
+              "suggestions": {
+                 "description": "An object that contains suggestions for behavior overrides for the declarative agent.",
```

Comment:
Update to " A JSON object that contains configuration settings for the suggestions feature. The object has a required disabled boolean property. If this property is set to true, the suggestions feature will be disabled. The default value is false"

Reply:
Updated